json_schema/meta/prism-credentials.json

```diff
@@ -216,6 +216,15 @@
                 "type": { "const": "array" },
                 "items": { "$ref": "#/$defs/field" }
               }
+            },
+            {
+              "title": "binary",
+              "type": "object",
+              "properties": {
+                "image": {
```

Comment:
mb, name if  more general ?
 Why this can't be audio, biometry, private key, etc ..  ?